src/models/Expense.model.js

```diff
@@ -1,9 +1,41 @@
 'use strict';
 
+const { DataTypes } = require('sequelize');
 const { sequelize } = require('../db.js');
 
 const Expense = sequelize.define(
-  // your code goes here
+  'Expense',
+  {
+    userId: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+    spentAt: {
+      type: DataTypes.DATE,
+      allowNull: false,
+    },
+    title: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    amount: {
+      type: DataTypes.FLOAT,
+      allowNull: false,
+    },
+    category: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    note: {
+      type: DataTypes.STRING,
+      allowNull: false,
```

Comment:
The `note` field is set to `allowNull: false`, which means it is required. If the `note` field is optional, consider setting `allowNull: true` or providing a default value to avoid validation errors when creating or updating an expense without a note.